SpatialGDK/Documentation/content/get-started/gdk-template.md

```diff
@@ -0,0 +1,150 @@
+
+If you are ready to start developing your own game with the GDK, follow the steps below. 
+
+### Terms used on this page
+
+* `<Game>` - The directory that contains your project’s .uproject file and Source folder.
+* `<ProjectRoot>` - The directory that contains your `<Game>` directory.
+* `<YourProject>` - The name of your project and .uproject file (for example, `\<Game>\YourProject.uproject`).
+
+### Create a new project using the Starter Template
+
+After [building the Unreal Engine fork]({{urlRoot}}/content/get-started/build-unreal-fork), in **File Explorer**, navigate to `UnrealEngine\Engine\Binaries\Win64`and double-click UE4Editor.exe to open the Unreal Editor. 
+
+1. In the [Project Browser](https://docs.unrealengine.com/en-us/Engine/Basics/Projects/Browser) window, select the **New Project** tab and then the **C++ tab**. 
+2. In this tab, select **SpatialOS GDK Starter**. 
+3. In the **Folder** field, choose a suitable directory for your project.
+4. In the **Name** field, enter a project name of your choice.
+5. Select **Create Project**.
+
+**Note:** When you create a project, the Unreal Engine automatically creates a directory named after the project name you entered. This page uses `YourProject` as an example project name.
+
+![The Unreal Engine Project Browser]({{assetRoot}}assets/set-up-template/template-project-browser.png)
+
+*Image: The Unreal Engine Project Browser, with the project file path and project name highlighted.*
+
+After you have selected **Create Project**, the Unreal Engine generates the necessary project files and directories, it then closes the Editor and automatically opens Visual Studio.
+
+### Clone the GDK
+
+Now you need to clone the SpatialOS GDK for Unreal into your project. To do this: 
+
+1. In **File Explorer**, navigate to the `<Game>` directory and create a `Plugins` folder in this directory.
+2. In a Git Bash terminal window, navigate to `<Game>\Plugins` and clone the [GDK for Unreal](https://github.com/spatialos/UnrealGDK) repository by running either:
+    * (HTTPS) `git clone https://github.com/spatialos/UnrealGDK.git`
+    * (SSH) `git clone git@github.com:spatialos/UnrealGDK.git`
+
+**Note:** You need to ensure that the root directory of the GDK for Unreal repository is called `UnrealGDK` so the file path is: `<Game>\Plugins\UnrealGDK\...`
+
+### Build the dependencies 
+
+To use the Starter Template, you must build the GDK for Unreal module dependencies and then add the GDK to your project. To do this: 
+
+1. Open **File Explorer**, navigate to the root directory of the GDK for Unreal repository (`<Game>\Plugins\UnrealGDK\...`), and double-click **`Setup.bat`**. If you haven't already signed into your SpatialOS account, the SpatialOS developer website may prompt you to sign in.
+1. In **File Explorer**, navigate to your <Game> directory and double-click **`<YourProject>`.sln** to open it with Visual Studio.
```

Comment:
I believe @m-samiec mentioned that before we build the project, we need to regenerate the project files since we have run the 'Setup.bat' script and altered the project file structure. To do this we would need to right click on the <YourProject>.uproject and click 'Generate Visual Studio Project files'. Also, Visual studio is automatically opened after the project is created, we may need to specify to the user to close visual studio before running 'Setup.bat' and regenerating visual studio project files?

Reply:
Feels like we're definitely missing a step somewhere here. We probably need to tell users to close VS as part of the `Create a new project` step, but we also need to tell them why VS has opened in the first place. When closing VS I get prompted to save my changes, presumably I have to save the changes for the rest of the setup to work. We need to make this clear.

Reply:
Yes running `Generate Visual Studio Project files` is highly recommended as otherwise the VS solution won't contain the new plugin files.

Reply:
```suggestion
1. Open **File Explorer**, navigate to the root directory of the GDK for Unreal repository (`<Game>\Plugins\UnrealGDK\...`), and double-click **`Setup.bat`**. If you haven't already signed into your SpatialOS account, the SpatialOS developer website may prompt you to sign in. 
1. In **File Explorer**, navigate to your `<Game>` directory, right-click `<YourProject>`.uproject and select Generate Visual Studio Project files.
1. In the same directory, double-click **`<YourProject>`.sln** to open it with Visual Studio.
1. In the Solution Explorer window, right-click on **`<YourProject>`** and select **Build**.
1. When Visual Studio has finished building your project, right-click on **`<YourProject>`** and select **Set as StartUp Project**.
1. Press F5 on your keyboard or select **Local Windows Debugger** in the Visual Studio toolbar to open your project in the Unreal Editor.<br/>
```
New text.

Reply:
<3